drk-ontology.ttl

```diff
@@ -1,20 +1,339 @@
+# This means: If a place allows people to bring Dogs and/or Service Animals, it can be inferred that it allows Assistance Dogs and Guide Dogs as well. If a place allows people to bring Assistance Dogs, it can be inferred that it allows Guide Dogs as well. If a place does not allow people to bring Guide Dogs, it can be inferred that it does not allow Dogs, Service Animals, and Assistance Dogs either.
+drk:AnimalPolicy
+    a              owl:Class ;
+    rdfs:label     "Animal Policy Class"@en ;
+    rdfs:comment   "A class that describes the policies of a place concerning bringing animals along."@en ;
+    dcterms:source <https://sozialhelden.github.io/a11yjson/describing-objects/0-model/> .
+
+# The drk:allowsDogs property describes if the place allows visitors to bring dogs in general.
+drk:allowsDogs
+    a              rdf:Property, owl:DatatypeProperty ;
+    rdfs:domain    drk:AnimalPolicy ;
+    rdfs:range     schema:Boolean ;
+    rdfs:label     "Allows Dogs"@en ;
+    rdfs:comment   "A property that describes if the place allows visitors to bring dogs in general."@en ;
+    dcterms:source <https://sozialhelden.github.io/a11yjson/describing-objects/0-model/> .
+
+# The drk:allowsServiceAnimals property describes if the place allows visitors to bring service animals along.
+# A Service Animal is an animal that has been trained to assist a disabled person (source: https://en.wikipedia.org/wiki/Service_animal).
+# While dogs are the most common service animals, other animals (like miniature horses, ferrets, parrots, and monkeys) may also serve in this role, depending upon regional regulations.
+drk:allowsServiceAnimals
+    a              rdf:Property, owl:DatatypeProperty ;
+    rdfs:domain    drk:AnimalPolicy ;
+    rdfs:range     schema:Boolean ;
+    rdfs:label     "Allows Service Animals"@en ;
+    rdfs:comment   "A property that describes if the place allows visitors to bring service animals along. A Service Animal is an animal that has been trained to assist a disabled person (source: https://en.wikipedia.org/wiki/Service_animal). While dogs are the most common service animals, other animals (like miniature horses, ferrets, parrots, and monkeys) may also serve in this role, depending upon regional regulations."@en ;
+    dcterms:source <https://sozialhelden.github.io/a11yjson/describing-objects/0-model/> .
+
+# The drk:allowsAssistanceDogs property describes if the place allows visitors to bring assistance/service dogs.
+# An assistance/service dog is a dog that receives specialized training to aid an individual with a disability in navigating everyday life (source: https://en.wikipedia.org/wiki/Assistance_dog).
+# Assistance/Service dogs are a specific category of service animals.
+drk:allowsAssistanceDogs
+    a                  rdf:Property, owl:DatatypeProperty ;
+    rdfs:subPropertyOf drk:allowsDogs, drk:allowsServiceAnimals ;
+    rdfs:domain        drk:AnimalPolicy ;
+    rdfs:range         schema:Boolean ;
```

Comment:
Yes, schema.org defines its own [datatypes](https://schema.org/DataType).
Also, as you might know already, schema.org does not use `rdfs:domain` and `rdfs:range` properties, but defines its own properties: `schema:domainIncludes` and `schema:rangeIncludes`, which are semantically different. For example, `schema:rangeIncludes` is used to indicate suggested types for values, instead of strict type enforcement.

I've now updated the ontology to use stricter datatypes:
- Replaced `schema:Boolean` with `xsd:boolean`
- Replaced `schema:Integer` with `xsd:nonNegativeInteger`